app/main/views/frameworks.py

```diff
@@ -11,3 +12,18 @@ def list_frameworks():
     return jsonify(
         frameworks=[f.serialize() for f in frameworks]
     )
+
+
+@main.route('/frameworks/g-cloud-7/stats', methods=['GET'])
+def get_framework_stats():
+
+    seven_days_ago = datetime.datetime.now() + datetime.timedelta(-7)
+
+    return str({
+        'drafts': DraftService.query.filter(DraftService.status == "not-submitted").count(),
```

Comment:
Drafts should probably be filtered by framework. We only have G7 drafts right now, but this might change if service edits go live before the end of SSP.